src/pages/OrgRegister.tsx

```diff
@@ -1,91 +1,121 @@
+/* eslint-disable */
 import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
 import { Link } from 'react-router-dom';
-import Logo from '../assets/logo.svg';
-import { IoPerson, IoMail, IoCreate } from 'react-icons/io5'
+import { FaRegUser, FaRegEnvelope, FaPen } from 'react-icons/fa';
+import { useForm } from 'react-hook-form';
 
 export default function OrgRegister() {
+  const { t } = useTranslation();
   const [name, setName] = useState("")
 
   const handleChange = (value: string) => {
     setName(value)
-    
   }
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    reset
+  }: any= useForm();
+  const onSubmit = (data: any) => {
+    console.log(data);
```

Comment:
There's a merged PR  with a hook that prevents `console`  messages on push. I'm wondering why it did not catch this one.